suite-native/intl/src/en.ts

```diff
@@ -765,8 +765,8 @@ export const en = {
             },
         },
         bluetooth: {
-            title: 'Bluetooth',
-            content: 'Connect to Trezor Suite via Bluetooth.',
+            title: 'Unpair Trezor',
+            content: 'Unpair your Trezor from this device.',
```

Comment:
typo. there should not be the dot. 

<img width="440" height="125" alt="Screenshot 2025-08-26 at 10 18 56" src="https://github.com/user-attachments/assets/3e6f6e51-89df-4f79-b9bd-5d2608cc4e2c" />